core/src/main/java/brooklyn/entity/basic/EntityPredicates.java

```diff
@@ -156,6 +156,36 @@ public String toString() {
             return "applicationIdSatisfies("+condition+")";
         }
     }
+    
+    public static Predicate<Entity> classNameEquals(final String name) {
```

Comment:
Perhaps should be `typeNameEquals` to better match `input.getEntityType().getName()`, which is subsequently called. This (normally?) returns the entity's interface name (rather than the implementation class name), which is another reason `typeNameEquals` would be clearer.